### ISSUE TITLE
Update .github/CONTRIBUTING.md with instructions for linting and testing

### ISSUE THREAD

**Issue by kalegd** [kalegd/DigitalBacon#27]
👋 Hello new potential contributors (current contributors welcome as well),

It would be great for `npm run lint` and `npm test` to be listed as commands people can run to verify a consistent coding style across the project and prevent regressions. I think the best place for this would be in the Next Steps section between the `npm start` details and `npm run build` details of CONTRIBUTING.md ([see here](https://github.com/kalegd/DigitalBacon/blob/b5eaafb9dc324f7f33202cd01c02a8ef1ef06624/.github/CONTRIBUTING.md?plain=1#L30))

**Comment by kaigidwani** [comment]
Hello! Can I contribute to this?

**Comment by kalegd** [comment]
Sure thing! I've gone ahead and assigned you to this issue @kaigidwani

**Comment by kalegd** [comment]
As it's been over a week without any updates, I'm going to remove @kaigidwani from the assignee. This issue is still up for grabs

**Comment by kaigidwani** [comment]
Hey! Really sorry about that! I got pretty swamped with work right after this and had to prioritize some other projects. If you'd still like my help I've got a draft here if you'd like to give it a look-over? I tried to replicate the style of the rest of the documentation, but let me know what you think and if there's anything I should change.

To verify that you are using a consistent coding style with the rest of the project, run:

npm run lint

To run a test of the project to prevent regressions, run:

npm test
